### ISSUE TITLE
Ticking player crash with FireNBlood

### ISSUE THREAD

**Issue by KrystalTheFox** [Polarice3/FireNBlood#3]
Hello. I recently started using FireNBlood on my modded Minecraft and while I was playing on one of my worlds, I got a ticking player crash and I was not sure what caused it. I first reported the issue to the forge website and they told me that it was an issue with FireNBlood and to report it to the mod author.

Forge Version 1.16.5 36.2.28
Mod Version: Latest
Crash report: https://pastebin.com/v9Dk4c1K
Log: https://pastebin.com/FrtdQ5FN

**Comment by Polarice3** [comment]
Did you press the "Open Wand Focus Slot" or the "Open Bag and Wand" key when holding anything but a Dark Wand/Staff? If so, that's the reason for the crash! The next update will have that crash fixed! Sorry about that!

**Comment by KrystalTheFox** [comment]
> Did you press the "Open Wand Focus Slot" or the "Open Bag and Wand" key when holding anything but a Dark Wand/Staff? If so, that's the reason for the crash! The next update will have that crash fixed! Sorry about that!

Nope. The last thing I did before the game crashed was throwing a trident with channeling and loyalty at one of the mobs during a thunderstorm.

**Comment by Polarice3** [comment]
Try updating the mod to 1.8.0, and see if the crash still persists.

**Comment by KrystalTheFox** [comment]
I updated to 1.8.2 and now im getting an initializing game crash
Crash Report: https://pastebin.com/Wi3MpN0W

**Comment by Polarice3** [comment]
It seems like it's a problem with Wolves with Armor.  Try updating that mod.

**Comment by KrystalTheFox** [comment]
> It seems like it's a problem with Wolves with Armor. Try updating that mod.

Updating that mod didnt fix it the crash. When I removed wolves with armor, It give me a mod loading error crash

Latest crash: https://pastebin.com/FN7y1MSX

**Comment by Polarice3** [comment]
I just look at your previous crash, the one with the Wolves with Armor, and found this: https://github.com/BloCamLimb/ModernUI/issues/51

It seems that it might be the version of Java you're playing on might be the cause of the crash. Try updating your java from 1.8.0_51 to 1.8.0_261 or above.

**Comment by KrystalTheFox** [comment]
I still get a mod loading error. even without wolves wear armor. Im on java 16 now

**Comment by Polarice3** [comment]
Try using 1.8.0_261 or above. Some 1.16.5 mods require Java 8 to function and not any other version of Java.

**Comment by KrystalTheFox** [comment]
> Try using 1.8.0_261 or above. Some 1.16.5 mods require Java 8 to function and not any other version of Java.

that Still didnt fix it.

**Comment by Polarice3** [comment]
Do you have Curio installed? If not, try installing it to your modpack. 

Also, the mod is discontinued, so support isn't maintained for the foreseeable future! Sorry.
